app/components/organization/Pipeline/bar.js

```diff
@@ -105,3 +112,13 @@ export default class Bar extends React.Component {
     }
   }
 }
+
+export default Relay.createContainer(Bar, {
```

Comment:
There’s only one `getFragment` call for _all_ the `Bar` instances, so that shouldn’t be a problem. As for “too many containers” I guess I can’t say. I suspect there’s no “Relay-friendly” way to make it cleaner, though.